Windows.bat

```diff
@@ -0,0 +1,34 @@
+# Download Nikto-Master and change the name to nikto_cpy
+@ECHO OFF
+@ECHO +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+@ECHO ++++++     +++    +++++     +++     +++    +++++++++      +++++++
+@ECHO +++ +++    +++     +++      +++   +++      +++++++++    ++++   ++++
+@ECHO +++  +++   +++     +++      ++++++            +++      ++++     ++++
+@ECHO +++   +++  +++     +++      ++++++            +++      ++++     ++++
+@ECHO +++    +++ +++     +++      +++   +++         +++       ++++   ++++
+@ECHO +++     ++++++    +++++     +++     +++       +++         +++++++
+
+@ECHO +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+@ECHO RUNNING UPDATE...
+@ECHO -----------------------------------
+git clone https://github.com/sullo/nikto.git D:\Nikto
+cd D:\nikto_cpy
```

Comment:
You cloned it into d:\Nikto then change into d:\nikto_cpy, that won't work

Reply:
Hello Mr.Sullo
The concept is to make the nikto_cpy update
and **git clone https://github.com/sullo/nikto.git** **D:\Nikto** should be a temporary location where after CD the nikto_cpy it pulls from the .git along with the update. I am a newbie and i'm learning. aslo i tried gitpull D:\Nikto to nikto_cpy, came with Access Denied.
So I was baffled and return to "run without error" concept. 
Sorry

Reply:
git clone https://github.com/sullo/nikto.git D:\Nikto
cd D:\Nikto
gitpull D:\nikto_cpy